ingestion_tools/scripts/tests/fixtures/dataset/invalid_http_keyphoto.yaml

```diff
@@ -0,0 +1,33 @@
+dataset_keyphotos:
```

Comment:
Todo: Update the tests to use the same config can be used with dynamic update to the key_photo entity.